nailgun/entities.py

```diff
@@ -6068,6 +6068,7 @@ def __init__(self, server_config=None, **kwargs):
             ),
             'full_path': entity_fields.StringField(),
             'gpg_key': entity_fields.OneToOneField(ContentCredential),
+            'ignorable_content': entity_fields.ListField(),
```

Comment:
Is there a fixed choice list to require here?

Reply:
I will take a look. Thanks.

Reply:
> List of content units to ignore while syncing a yum repository. Must be subset of rpm,drpm,srpm,distribution,erratum

Reply:
I was thinking to add a validator to ensure the values on that list are on that subset, but looks like the `ListtField` does not support `choices` or any other kind of validator. So nm.